ansible_ai_connect/ai/api/exceptions.py

```diff
@@ -183,3 +183,58 @@ class FeedbackInternalServerException(BaseWisdomAPIException):
     status_code = 500
     default_code = "error__feedback_internal_server"
     default_detail = "Failed to send feedback"
+
+
+class ChatbotNotEnabledException(BaseWisdomAPIException):
+    status_code = 503
+    default_code = "error__chatbot_not_enabled"
+    default_detail = "Chatbot is not enabled"
+
+
+class ChatbotInvalidRequestException(BaseWisdomAPIException):
+    status_code = 400
+    default_code = "error__chatbot_invalid_request"
+    default_detail = "Invalid request"
+
+
+class ChatbotInvalidResponseException(BaseWisdomAPIException):
+    status_code = 500
+    default_code = "error__chatbot_invalid_response"
+    default_detail = "Invalid response"
+
+
+class ChatbotUnauthorizedException(BaseWisdomAPIException):
+    """
+    Since this exception is on the authentication between
+    AI Connect server and Chatbot, which is a backend service,
+    status_code 503 (Service Unavailable) is returned to
+    client instead of 401.
+    """
+
+    status_code = 503
+    default_code = "error__chatbot_unauthorized"
+    default_detail = "Unauthorized"
+
+
+class ChatbotForbiddenException(BaseWisdomAPIException):
```

Comment:
Picky, you could extend `WisdomBadRequest`

https://github.com/ansible/ansible-ai-connect-service/blob/10a81f71d219f30fa749797c5f817ad933accc52/ansible_ai_connect/ai/api/exceptions.py#L73